### DISCUSSION TITLE
3 Accesspoints mit Windows Controller Software

### DISCUSSION THREAD

**Discussion by Sasch600xt** [elueckel/Unifi-Toolbox#72] [Q&A] [answered]
Hallo zusammen, bevor ich mich zu tief in die Unifi Toolbox einlese, habe ich nur eine kurze Frage:

Ich habe 3 APs und die Controller Software für Windows installiert.
Kann ich damit die Toolbox hier schon verwenden, oder brauche ich mehr Hardware, bzw einen Hardware Controller statt "nur" der Windows Software (läuft 24/7)

Vielen lieben Dank für die Hilfe :)
Sascha

**Comment by Brovning** [comment, marked as answer]
Hallo Sascha,

ja, das geht auch mit einem selbst gehostetem Controller (egal ob Docker, Windows, Linux,...).
Sofern die Auswahl DreamMachine / CloudKey (=Controller) zur Verfügung steht, den "CloudKey" auswählen.
Ist keine Auswahl verfügbar geht es glaube ich nur für DreamMachine

**Comment by elueckel** [comment]
Wichtig ist auch das du Symcon als Middleware hast. Das Modul funktioniert nur damit.

**Comment by Sasch600xt** [comment]
Danke für die Antworten. Was bedeutet  "Symcon als Middleware" ? Ich habe Symcon auf dem gleichen PC  wie den Controller installiert.
Also könnte ich als IP Adresse des Controllers in Symcon sogar 127.0.0.1 nehmen, oder ?

Liebe Grüße
Sascha

**Comment by Sasch600xt** [comment]
![image](https://github.com/elueckel/Unifi-Toolbox/assets/25459028/7fdfc0ad-3af7-47e9-b559-f198120df20e)
Das hier ist meine Auswahl in den Modulen.

**Reply by Brovning** [reply]
Du musst für den gehosteten Controller den "CloudKey" auswählen.
Habe meine Antwort oben entsprechend editiert.

**Comment by Sasch600xt** [comment]
Ok, danke.
Hab mal alle möglichen Versionen durchprobiert und wollte mal ein UniFi WLAN Device überwachen.
Eventuell hat das noch mit dem "Symcon als Middleware" zu tun, was ich noch nicht ganz verstehe,

Aber schonmal 1000 Dank für die Antworten :)

![image](https://github.com/elueckel/Unifi-Toolbox/assets/25459028/47ec9181-8dd7-4472-bda8-9b89edc93c61)

**Reply by Brovning** [reply]
Du kannst den Anmeldevorgang testen, indem du unten auf "Test Anmeldung am Controller" klickst.
Probier' doch mal deine richtige IP Adresse aus.
Mit IPS als Middleware ist gemeint, dass dieses Modul nur funktioniert, wenn du IP Symcon nutzt, aber das ist ja der Fall.

Was wird im Debug Log angezeigt?

**Comment by Sasch600xt** [comment]
in dem Zusammenhang:
Ist das ein kleiner Tippfehler ? Müsste das nicht "generic" Device heißen ?
Na wenigstens finde ich kleine Tippfehler :)

**Comment by Sasch600xt** [comment]
Muss ich im UniFi Controller eventuell noch irgendetwas freischalten?

![image](https://github.com/elueckel/Unifi-Toolbox/assets/25459028/4252ef81-57dc-4d56-8c51-31df8203e802)

**Comment by elueckel** [comment]
Also 404 heist „nicht erreichbar“ … du musst die IP verwenden und den richtigen Port. Dann sollte eine Fehler kommen das der User nicht passt, aber aktuell kommst du nicht auf den Port.

**Comment by Sasch600xt** [comment]
Au Backe !!!! Den UniFi Controller habe ich schon seit Jahren am laufen......damals hatte ich den Port geändert, was ich völlig vergessen habe !!!
Port ist nun richtig, und er sagt "Site Falsch" Unter Default habe ich wirklich etwas anderes, meine neue Seite heißt "Home" Ich habe es auch mal kleingeschrieben probiert, aber der Fehler bleibt....
![image](https://github.com/elueckel/Unifi-Toolbox/assets/25459028/5944d04f-fc72-42a9-aa5d-1856cdc08912)

![image](https://github.com/elueckel/Unifi-Toolbox/assets/25459028/a3b1e80e-ed09-477a-9e9b-bbe448ba9ac8)

Leider muss ich nun arbeiten gehen, ich versuche heute Abend hier noch mal weiterzumachen.
Tut mir sehr leid, wegen dem falschen Port :( Wollte deine Zeit nicht verschwenden.

Vielen Dank für deine Hilfe bis hier hin :)
Liebe Grüße
Sascha

**Reply by Brovning** [reply]
Dann bitte mal auf "Name der Site prüfen" klicken.

**Comment by Sasch600xt** [comment]
Das ist ja irre :) Aber wenn ich den Namen, den Dein Modul gefunden hat, benutze, geht es wohl :)
Vielen 1000 Dank für deine Hilfe :)

![image](https://github.com/elueckel/Unifi-Toolbox/assets/25459028/125a6b71-c7af-4e90-a095-71a6f555b1eb)

**Reply by Brovning** [reply]
Freut mich, dass es nun funktioniert.

**Reply by Sasch600xt** [reply]
Ja vielen Dank für dein "Nicht aufgeben" :)
